pkg/plugins/backendplugin/log_wrapper.go

```diff
@@ -60,7 +76,18 @@ func (lw logWrapper) With(args ...interface{}) hclog.Logger {
 	return logWrapper{Logger: lw.Logger.New(args...)}
 }
 func (lw logWrapper) Named(name string) hclog.Logger {
-	return logWrapper{Logger: lw.Logger.New()}
+	discardLogs := false
+	if name == "stdio" {
+		// discard logs from stdio hashicorp/go-plugin gRPC service since
+		// it's not enabled/in use per default.
+		// discard debug log of "waiting for stdio data".
+		// discard warn log of "received EOF, stopping recv loop".
+		discardLogs = true
```

Comment:
Another option would be to return a Noop logger here that doesn't log anything. This is fine as well. If we stick to this solution I think it would be cleaner to invert the if statements to avoid the early return and just wrap the log statement.

Reply:
Good idea. Will try it out